gulpfile.babel.js

```diff
@@ -22,7 +22,7 @@ if (path.win32 === path) {
   srcEx = /(packages\\[^\\]+)\\src\\/
   libFragment = '$1\\lib\\'
 } else {
-  srcEx = /(packages\/[^\/]+)\/src\//
+  srcEx = /(packages\/[^/]+)\/src\//
```

Comment:
since you are in a character class, the slash is not special but interpreted literally. The escape can be omitted